restart.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
+cd "$(dirname "$0")"
 
-# Pull and up containers
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml pull
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml up --force-recreate --detach --remove-orphans
```

Comment:
So you have made a `restart-dev.sh` script or?

Reply:
Don't see why we should need a `dev` script. Should be handled by our docker setup?